load-testing/stress-test.ts

```
@@ -0,0 +1,229 @@
+        await setTimeout(getRandomInt(scenario.maxWait, scenario.minWait));
+      }
+
+      console.log(`Client '${socket.id}' finished all actions for '${roomID}'`);
+      // tear down
+      clearInterval(idleStateInterval);
+      clearInterval(mouseLocationInterval);
+      socket.disconnect();
+
+      resolve();
+    });
+  });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)